plugins/Table2/ui/Table2.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+    <v-data-table :headers="headers" :items="table" class="elevation-1" item-key='Component' :expand.sync="expanded" show-expand :item-class="itemClass">
+        <template v-slot:top>
+            <v-toolbar flat>
+                <v-toolbar-title>T.A.B.L.E.2.</v-toolbar-title>
+                <v-divider class="mx-4" inset vertical></v-divider>
+                <v-toolbar-description>{{ subtitle }}</v-toolbar-description>
```

Comment:
`<v-toolbar-description>` is not a valid component. Maybe replace it with a simple `span`?